app/components/stepper.js

```diff
@@ -9,9 +9,13 @@ import { JOIN_URL } from '../constants/apis';
 export default class StepperComponent extends Component {
   @service login;
   @service toast;
+  @service router;
   @tracked preValid = false;
   @tracked isValid = JSON.parse(localStorage.getItem('isValid')) ?? false;
-  @tracked currentStep = +localStorage.getItem('currentStep') ?? 0;
+  @tracked currentStep =
+    +localStorage.getItem('currentStep') ??
+    +new URLSearchParams(window.location.search).get('step') ??
```

Comment:
Do we really need `window` or it can be done with `router-service`, please confirm.
Also check these docs RouterService method [currentRoute](https://api.emberjs.com/ember/4.10/classes/RouterService/properties/currentRoute?anchor=currentRoute&show=inherited%2Cprotected%2Cprivate) 

[RouteInfo](https://api.emberjs.com/ember/4.10/classes/RouteInfo)

Reply:
see [this](https://github.com/Real-Dev-Squad/website-www/pull/408#discussion_r1125305308)